bin/dynamodb-admin.js

```diff
@@ -35,12 +41,13 @@ parser.add_argument('-p', '--port', {
 const args = parser.parse_args()
 
 const app = createServer()
+const host = process.env.HOST || args.host
 const port = process.env.PORT || args.port
-const server = app.listen(port)
+const server = app.listen(port, host)
 server.on('listening', () => {
   const address = server.address()
-  const url = `http://localhost:${address.port}`
-  console.log(`  dynamodb-admin listening on ${url} (alternatively http://0.0.0.0:${address.port})`)
+  const url = `http://${address.address}:${address.port}`
+  console.log(`  dynamodb-admin listening on ${url})`)
```

Comment:
You left stray parenthesis there but no big deal.

And this also changes behavior in the default case. Previously it would show:

```
dynamodb-admin listening on http://localhost:8001 (alternatively http://0.0.0.0:8001)
```

Now it will show:

```
dynamodb-admin listening on http://127.0.0.1:8001)                    
```

So both the `localhost` and `0.0.0.0` variants are now missing. The latter at least was kinda useful, letting the user know that it's running on a broadcast address.

Reply:
And also, I believe, this would break usage within our docker container since it needs to run on broadcast address there to be exposed.

Reply:
Created follow-up fix #208

Reply:
Woops, thanks for creating fix